packages/@webex/plugin-meetings/test/unit/spec/media/MediaConnectionAwaiter.ts

```diff
@@ -99,9 +151,10 @@ describe('MediaConnectionAwaiter', () => {
       assert.equal(promiseRejected, false);
 
       // check the right listener was registered
-      assert.calledTwice(mockMC.on);
-      assert.equal(mockMC.on.getCall(0).args[0], Event.CONNECTION_STATE_CHANGED);
-      assert.equal(mockMC.on.getCall(1).args[0], Event.ICE_GATHERING_STATE_CHANGED);
+      assert.calledThrice(mockMC.on);
+      assert.equal(mockMC.on.getCall(0).args[0], Event.PEER_CONNECTION_STATE_CHANGED);
+      assert.equal(mockMC.on.getCall(1).args[0], Event.ICE_CONNECTION_STATE_CHANGED);
+      assert.equal(mockMC.on.getCall(2).args[0], Event.ICE_GATHERING_STATE_CHANGED);
 
       mockMC.getConnectionState.returns(ConnectionState.Connected);
```

Comment:
Question, why did we not have to call the listeners here like we did in the earlier tests?

Reply:
@k-wasniowski But if the connected state and connection state event hasn't been received, how does the promise get resolved correctly?

Reply:
It will be resolved by timeout. This test is forwarding clock in line 161, which will result in triggering onTimeout callback.